### PR TITLE
[Data Factory] Add Switch + AzureMLExecutePipeline activity, Update AzureMLService + HDInsightPig

### PR DESCRIPTION
**Pull request by mablumen**

### CODE REVIEW

**Review by isra-fel — CHANGES_REQUESTED** [review]
Hi @mablumen , please generate SDK code by running `generate.ps1` and track the metadata file. [(details)](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#generating-net-sdk)
Besides, please put a link to your spec review PR in the comments.
THanks

**Comment by mablumen** [comment]
@isra-fel Should I make a new PR. Or can I execute generate.ps1 on top of these changes?

**Comment by isra-fel** [comment]
> @isra-fel Should I make a new PR. Or can I execute generate.ps1 on top of these changes?

It calls autorest under the hood, so just run it on top of your changes.
Thanks

**Comment by mablumen** [comment]
Hi @isra-fel @erich-wang 
I tried running generate.ps1 and get following error:
```Start-AutoRestCodeGeneration : The term 'Start-AutoRestCodeGeneration' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a
path was included, verify that the path is correct and try again.```

So I tried running msbuild eng/mgmt.proj /t:Util /p:UtilityName=InstallPsModules and get following error:
```C:\Git\azure-sdk-for-net\eng\mgmt\Directory.Build.Mgmt.props(10,3): error MSB4019: The imported project "C:\Git\azure-sdk-for-net\eng\mgmt\CI.Bootstrap.targets" was not found. Confirm that the path in t
he <Import> declaration is correct, and that the file exists on disk. [C:\Git\azure-sdk-for-net\eng\mgmt.proj]```

**Comment by mablumen** [comment]
Hi @isra-fel @erich-wang
I've resolve the above issue and am now able to run `generate.ps1`

However I'm getting this new error when I run the script:
`npm ERR! Error: EPERM: operation not permitted, rename 'C:\Users\mablumen\AppData\Roaming\npm\node_modules\autorest'
-> 'C:\Users\mablumen\AppData\Roaming\npm\node_modules\.autorest.DELETE'
npm ERR!  { [Error: EPERM: operation not permitted, rename
'C:\Users\mablumen\AppData\Roaming\npm\node_modules\autorest' ->
'C:\Users\mablumen\AppData\Roaming\npm\node_modules\.autorest.DELETE']`

Any thoughts?

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## Current version
 ###  Feature Additions
+- Add Switch activity
```
Comment (LijuanZ) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: Also add "- Added new AzureMLExecutePipeline activity and AzureMLService linked service type

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/tests/JsonSamples/PipelineJsonSamples.cs [hunk 2]
```diff
@@ -1883,6 +1883,46 @@ public class PipelineJsonSamples : JsonSampleCollection<PipelineJsonSamples>
...
+                                }
+                            ]
+                        }
+                    ]
+                }
+            }
+        ]
+    }
+}
+";
```
Comment (LijuanZ) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/tests/JsonSamples/PipelineJsonSamples.cs]: Add sample for azureMLExecutePipeline activity, I'll will send you sample

**Comment by mablumen** [comment]
Hi @isra-fel @erich-wang the above issues were all fixed. Can you please review now?

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/tests/JsonSamples/LinkedServiceJsonSamples.cs [hunk 3]
```diff
@@ -246,6 +246,36 @@ public class LinkedServiceJsonSamples : JsonSampleCollection<LinkedServiceJsonSa
...
+        connectVia: {
+            referenceName : ""Connection1"",
+            type : ""IntegrationRuntimeReference""
+        },
+        typeProperties:
+        {
+            connectionString: {
+                value : ""fakeConnString"",
+                type : ""SecureString""
+            },
```
Comment (LijuanZ) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/tests/JsonSamples/LinkedServiceJsonSamples.cs]: Sorry..my back, please remove these lines

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj [hunk 4]
```diff
@@ -5,20 +5,14 @@
...
-* Added support for Data Flow:
-    - Added data flow object model and CRUD APIs
-    - Added data flow debug session APIs
-    - Added support for execute data flow activity type
-    - Added support for data flow properties in managed integration runtime
-- Added property queryTimeout to tabular sources
-- Added support for Cosmos DB SQL API dataset and copy source/sink
-- Add dedicated linkedservice/dataset/source for Azure File Storage;
-- Add dedicated linkedservice/dataset/source for GoogleCloudStorage;
+- Add Switch activity
```
Comment (hvermis) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Microsoft.Azure.Management.DataFactory.csproj]: Added

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 5]
```diff
@@ -2,6 +2,9 @@
 
 ## Current version
 ###  Feature Additions
```
Comment (hvermis) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: Change the version here too to 4.3.0. Make sure to leave the Current Version at the top of the file

File: sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md [hunk 6]
```diff
@@ -3,6 +3,12 @@
...
 
+## Version 4.3.0
+- Added Switch activity
```
Comment (hvermis) [inline sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/changelog.md]: feature additions

**Review by hvermis — APPROVED** [review]
Ready for the release.

**Review by isra-fel — APPROVED** [review]

**Pull request merged**